### PR TITLE
Cleaned up some code from lazygit

### PR DESCRIPTION
**Pull request by mjarkk**

### CODE REVIEW

File: coverage.txt [hunk 1]
```diff
@@ -1,87 +1,5 @@
 
 mode: atomic
-github.com/jesseduffield/lazydocker/pkg/commands/container.go:247.64,249.2 1 0
```
Comment (jesseduffield) [inline coverage.txt]: is this file supposed to be committed?

**Comment by jesseduffield** [comment]
LGTM

**Comment by jesseduffield** [comment]
although I'm still interested in knowing: should the coverage.txt file be committed @mjarkk ?

**Comment by mjarkk** [comment]
@jesseduffield I think it's a better idea to ignore the coverage file by default for cleaner commits.

File: coverage.txt (hunk 1 above)
Comment (mjarkk) [reply coverage.txt]: No but because i ran the ./test.sh it gave me this what i accidentally commited because of a `git add .`

**Pull request merged**